sklearn/model_selection/_search.py

```diff
@@ -234,8 +241,8 @@ def __iter__(self):
             if grid_size < self.n_iter:
                 raise ValueError(
                     "The total space of parameters %d is smaller "
-                    "than n_iter=%d." % (grid_size, self.n_iter)
-                    + " For exhaustive searches, use GridSearchCV.")
+                    "than n_iter=%d." % (grid_size, self.n_iter) +
+                    " For exhaustive searches, use GridSearchCV.")
```

Comment:
Could you please avoid changing lines that are not part of your PR?
(That will help speed up the review of the actual content of your PR, helping you to merge it sooner into scikit-learn :) )